andreas-antonopoulos/2018-10-07-andreas-antonopoulos-schnorr-signatures.es.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Andreas Antonopoulos - Firmas Schnorr (2018-10-07)
+transcript_by: Michael Folkson
```

Comment:
You forget `translation_by` you, you can add it but that is not mandatory

Reply:
Thank you, I have added **translation_by**